dewy.go

```diff
@@ -326,6 +328,14 @@ func newRegistry(urlstr string, preRelease bool, artifactName string) (registry.
 			PreRelease: preRelease,
 		}
 		return ghrelease.New(c)
+	case grpcr.Scheme:
+		// TODO: handle tls
```

Comment:
It needs to be designed in conjunction with the GitHub Release registry, so it is a TODO in this PR.